src/components/Admin/Feedback.vue

```diff
@@ -12,7 +12,9 @@
                 <q-item-section>
                   <q-item-label>{{item.issue}}</q-item-label>
                   <q-item-label caption>{{item.user?users[item.user].fullname:'no user id'}}</q-item-label>
-                  
+                  <q-item-label v-if="item.platformdata">
```

Comment:
Is this still needed or is this leftover debug code?

Reply:
This is new extra information that will show more information regarding the device data of the user who send feedback.